custom_components/versatile_thermostat/underlyings.py

```diff
@@ -214,6 +217,13 @@ def is_inversed(self):
         """Tells if the switch command should be inversed"""
         return self._thermostat.is_inversed
 
+    @overrides
+    def startup(self):
+        super().startup()
+        self._keep_alive.set_async_action(
+            self.turn_on if self.is_device_active else self.turn_off
+        )
```

Comment:
As per other comment, even though this piece of code always runs, it will do nothing if the `keep_alive` setting is unset or zero.

Reply:
I  have seen after that it is done below. You can forget this comment.